### PR TITLE
Fixed broken link in central dashboard

### PR DESCRIPTION
**Pull request by hbelmiro**
Resolves https://github.com/kubeflow/website/issues/3809

### CODE REVIEW

**Comment by hbelmiro** [comment]
@andreyvelich can you please take a look?

**Comment by hbelmiro** [comment]
@andreyvelich @terrytangyuan mind taking a look?

**Comment by terrytangyuan** [comment]
Can you tag someone familiar with dashboard to review?

File: netlify.toml [hunk 1]
```diff
@@ -27,7 +27,6 @@ package = "netlify-plugin-checklinks"
 
   # If a link contains these patterns it will not fail the build, but will be reported as a TODO in the build log.
   todoPatterns = [
-    "public/docs/components/central-dash/",
```
Comment (terrytangyuan) [inline netlify.toml:30]: Why removing this? I am not familiar with this file

Comment (hbelmiro) [reply netlify.toml:30]: This line adds the `public/docs/components/central-dash/` to the white list of [checklinks](https://github.com/Munter/netlify-plugin-checklinks). Broken links under those directories are ignored during the CI.
This PR fixes all the broken links under that directory. Therefore we can now remove the directory from the white list so the CI verifies for broken links there too.

**Pull request merged**